### PR TITLE
Tperps 895 display trade history

### PR DESCRIPTION
**Pull request by stevenyyan**
### Motivation

Remove dummy data from the trade history tab and fetch that from filled orders in the OME store

### Changes

- Display trade history
- Clean up tables

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -96,7 +97,9 @@ const App = ({ Component, pageProps }: AppProps) => {
                         <GraphProvider>
                             <FactoryStore>
                                 <TransactionStore>
-                                    <Component {...pageProps} />
+                                    <OMEStore>
```
Comment (dospore) [inline pages/_app.tsx]: The insurance page wont need the OMEStore. I recommend just using 
```
    const { filledOrders, refetchFilledOrders } = useUsersMatched(selectedTracer?.address ?? '', account ?? '');
```
as a standalone hook

File: pages/index.tsx [hunk 2]
```diff
@@ -18,15 +17,13 @@ const Trade: React.FC = styled(({ className }) => {
...
-                        </OrderStore>
-                    </InsuranceStore>
-                </OMEStore>
+                <InsuranceStore>
+                    {/*TODO: Enable basic trading*/}
+                    {/* <OrderStore>{advanced ? <Advanced /> : <Basic />}</OrderStore> */}
+                    <OrderStore>
+                        <Advanced />
+                    </OrderStore>
+                </InsuranceStore>
```
Comment (dospore) [inline pages/index.tsx]: Re add OMEStore here

**Review by dospore — CHANGES_REQUESTED** [review]
Made a few comments. Just use the individual useUsersMatched if you want the users orders, instead of fetching all current trades

File: components/Portfolio/Trading/TradeHistory/index.tsx [hunk 3]
```diff
@@ -1,96 +1,63 @@
...
+    useEffect(() => {
+        const tempOrders: any[] = [];
+        fetchedTracers.map((tracer: any) => {
+            if (allFilledOrders[tracer.address] !== []) {
+                tempOrders.push(allFilledOrders[tracer.address]);
+            }
+        });
+        tempOrders.sort((order1, order2) => (order1.timestamp < order2.timestamp && 1) || -1);
+        setOrderHistory(tempOrders);
+    }, [fetchedTracers]);
```
Comment (dospore) [inline components/Portfolio/Trading/TradeHistory/index.tsx]: Do you need to filter here? isnt it already filtered from the graph?

File: components/Portfolio/Trading/index.tsx [hunk 4]
```diff
@@ -10,17 +10,31 @@ import { FactoryContext, initialFactoryState } from '@context/FactoryContext';
...
+        for (const key of Object.keys(tracers)) {
+            tempTracers.push(tracers[key]);
+        }
+        setFetchedTracers(tempTracers);
+    }, 1000);
+
+    useEffect(() => {
+        fetchTracers;
+    });
+
```
Comment (dospore) [inline components/Portfolio/Trading/index.tsx:26]: Does this even call the function??

**Review by dospore — CHANGES_REQUESTED** [review]

File: components/Portfolio/Trading/index.tsx (hunk 4 above)
Comment (stevenyyan) [reply components/Portfolio/Trading/index.tsx:26]: yep, every second

File: components/Portfolio/Trading/TradeHistory/index.tsx (hunk 3 above)
Comment (stevenyyan) [reply components/Portfolio/Trading/TradeHistory/index.tsx]: yeah you're right, probably don't need to filter here

File: components/Portfolio/Trading/index.tsx (hunk 4 above)
Comment (dospore) [reply components/Portfolio/Trading/index.tsx:26]: i see. Also gets called whenever the component updates. Which means you are adding a lot of timeout functions will need to update this. But approved

**Review by dospore — APPROVED** [review]

**Pull request merged**